### PR TITLE
Add version: LinwoodCloud.Butterfly version 2.1.0

### PR DESCRIPTION
**Pull request by CodeDoctorDE**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/152979)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   EC6EFD33CCAD12C990BF16F58660A928E85F66DEDC5811CFA8219B7C5DD5160C
 Expected hash: fd15a9b63d1355de1dd96619ef25feeb9a1b17479999d2b8fbb6e50b3d4a8989
 
 5/10/2024 7:46:21 AM
 
 ****************************************************************************************
 Installer Validation
 ****************************************************************************************
 

(Automated response - build 880.)

File: manifests/l/LinwoodCloud/Butterfly/2.1.0/LinwoodCloud.Butterfly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ReleaseDate: 2024-05-10
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/LinwoodDev/Butterfly/releases/download/v2.1.0/linwood-butterfly-windows-setup-x86_64.exe
+  InstallerSha256: FD15A9B63D1355DE1DD96619EF25FEEB9A1B17479999D2B8FBB6E50B3D4A8989
```
Comment (stephengillie) [inline manifests/l/LinwoodCloud/Butterfly/2.1.0/LinwoodCloud.Butterfly.installer.yaml:19]:
```suggestion

```

(Automated response - build 880.)

File: manifests/l/LinwoodCloud/Butterfly/2.1.0/LinwoodCloud.Butterfly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/LinwoodDev/Butterfly/releases/download/v2.1.0/linwood-butterfly-windows-setup-x86_64.exe
+  InstallerSha256: FD15A9B63D1355DE1DD96619EF25FEEB9A1B17479999D2B8FBB6E50B3D4A8989
```
Comment (stephengillie) [inline manifests/l/LinwoodCloud/Butterfly/2.1.0/LinwoodCloud.Butterfly.installer.yaml:25]:
```suggestion

```

(Automated response - build 880.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-05-14 13:13:30.392 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.33135.0

(Automated response - build 881.)

**Comment by Trenly** [comment]
* #152555

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/pull/156805#issuecomment-2211069274

**Review by stephengillie — APPROVED** [review]

**Comment by CodeDoctorDE** [comment]
hmm 2.1.1 was already released, i will close this

**Comment by stephengillie** [comment]
I was in the process of validating, but okay. (It passed, but I hadn't got back around to this PR yet.)

**Comment by CodeDoctorDE** [comment]
Hmm yeah, but i think it would be better if this could be merged, https://github.com/microsoft/winget-pkgs/pull/157221. I can't reopen it but this is the latest stable that should be merged

**Pull request closed without merging**